cdap-ui/app/cdap/components/Reports/store/ReportsStore.js

```diff
@@ -65,13 +81,39 @@ const customizer = (state = defaultCustomizerState, action = defaultAction) => {
         ...state,
         [action.payload.type]: !state[action.payload.type]
       };
+    case ReportsActions.setSelections:
+      return {
+        ...state,
+        ...action.payload.selections
+      };
+    case ReportsActions.clearSelection:
     case ReportsActions.reset:
       return defaultCustomizerState;
     default:
       return state;
   }
 };
 
+const status = (state = defaultStatusState, action = defaultAction) => {
+  switch (action.type) {
+    case ReportsActions.setStatus:
+      return {
+        ...state,
+        selections: action.payload.selections
+      };
+    case ReportsActions.setSelections:
+      return {
+        ...state,
+        selections: action.payload.status
```

Comment:
should this be `action.payload.selections`? but if that's the case, what's the difference between `ReportsActions.setStatus` and `ReportsActions.setSelections`?

Reply:
there is no difference for this reducer, however setSelections will dispatch updates to other reducers